builder/steps/gen-dockerfile/contents/src/detect_setup.ts

```diff
@@ -140,7 +145,14 @@ export async function detectSetup(
   if (!(await fsview.exists(appYamlPath))) {
     throw new Error(`The file ${appYamlPath} does not exist`);
   }
-  const config = yaml.safeLoad(await fsview.read(appYamlPath));
+  const rawConfig = yaml.safeLoad(await fsview.read(appYamlPath));
```

Comment:
Isn't it possible to make this line
```ts
const config: YamlConfig|undefined = yaml.safeLoad(...);
```
, instead of typecasting later below?

Reply:
Yes, I could do that too.

Reply:
I've updated the code.